tinyssh_install

```diff
@@ -1,14 +1,27 @@
 #!/bin/bash
 
+KEY_DIR=/etc/tinyssh/sshkeydir
```

Comment:
I extracted this magic string because it was referenced too many times to my taste. There are other magic strings wandering about but this calls for another refactoring pass.